frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -221,4 +227,25 @@ describe('Kubevirt create VM using wizard', () => {
     },
     CLONE_VM_TIMEOUT_SECS,
   );
+
+  it('ICNV-5045 - dont let the user continue If PXE provision source is selected on a cluster without a NAD available', async () => {
+    deleteResources([multusNAD]);
+    const vm = new VMBuilder(getBasicVMBuilder())
+      .setFlavor(flavorConfigs.Tiny)
+      .setName(testName)
+      .setOS(OperatingSystem.FEDORA)
+      .setProvisionSource(ProvisionSource.PXE)
+      .setWorkload(Workload.DESKTOP)
+      .build();
+    const wizard = new Wizard();
+    await vm.navigateToListView();
+    await isLoaded();
+    await wizard.openWizard();
+    await wizard.processGeneralStep(vm.getData(), true);
+    await browser.wait(
+      waitForStringInElement(view.footerError, 'Please correct the following field: Boot Source.'),
+      1000,
+    );
+    createResources([multusNAD]);
```

Comment:
probably it is better to change beforeAll to beforeEach instead + the same for afterAll